fission/src/ui/panels/configuring/assembly-config/interfaces/ConfigureShotTrajectoryInterface.tsx

```diff
@@ -208,12 +208,12 @@ const ConfigureShotTrajectoryInterface: React.FC<ConfigEjectorProps> = ({ select
             <Button
                 value="Reset"
                 onClick={() => {
-                    if (transformGizmo) {
+                    if (ejectorMesh) {
                         const robotTransformation = JoltMat44_ThreeMatrix4(
                             World.PhysicsSystem.GetBody(selectedRobot.GetRootNodeId()!).GetWorldTransform()
                         )
-                        transformGizmo.mesh.position.setFromMatrixPosition(robotTransformation)
-                        transformGizmo.mesh.rotation.setFromRotationMatrix(robotTransformation)
+                        ejectorMesh.position.setFromMatrixPosition(robotTransformation)
+                        ejectorMesh.rotation.setFromRotationMatrix(robotTransformation)
```

Comment:
There's a lot of repetition of these two functions. Would it be worth creating a function to do that for you? There's already one in the gizmo class but that only works if the mesh is stored in the gizmo.